resources/migrations/20190205094615-catalogue-item-application-extra-tables.down.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE public.catalogue_item_application_free_comment_values (
```

Comment:
Remove the schema ("public." prefix). We cannot assume that what the user's default schema is.